exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java

```diff
@@ -262,17 +265,15 @@ private boolean waitAndAssertDrillbitCount(ClusterFixture cluster, int zkRefresh
   }
 
   private static void setupFile(int file_num) throws Exception {
-    final String file = "employee"+file_num+".json";
-    final Path path = dirTestWatcher.getRootDir().toPath().resolve(file);
-    try(PrintWriter out = new PrintWriter(new BufferedWriter(new FileWriter(path.toFile(), true)))) {
+    String file = "employee" + file_num + ".json";
+    Path path = dirTestWatcher.getRootDir().toPath().resolve(file);
+    try (PrintWriter out = new PrintWriter(new BufferedWriter(new FileWriter(path.toFile(), true)))) {
```

Comment:
I realize the code here is original; but it might be a bit cleaner to put the data in a resource file than in Java.